clusterous/cluster.py

```diff
@@ -1453,7 +1453,7 @@ def sync_get(self, local_path, remote_path):
         stdin, stdout, stderr = ssh.exec_command(cmd)
         output_content = stdout.read()
         if 'cannot access' in stderr.read():
-            message = "Error: Folder '{0}' does not exists.".format(remote_path)
+            message = "Error: Folder '{0}' does not exist.".format(remote_path)
```

Comment:
As per the list, could you also remove the "Error:" prefix from the error messages?